product_pricelist_revision/models/pricelist.py

```diff
@@ -8,7 +8,7 @@ class ProductPricelistItem(models.Model):
     _inherit = "product.pricelist.item"
     _rec_names_search = ["name"]  # Initialize here to be able to extend
 
-    name = fields.Char(store=True)
+    name = fields.Char(store=True, compute_sudo=True)
```

Comment:
@gdgellatly Do you have access errors ? Because it seems name computation does not involve objects that could have different access rights than pricelist.item itself

Reply:
@rousseldenis No this is solely because a multi now combines a stored and a non stored field. Oh, looks like I got the wrong field. I picked the one to changed to stored, I need to declare the other one as compute_sudo.